subsys/net/lib/nrf_cloud/src/nrf_cloud_fota.c

```diff
@@ -795,6 +795,7 @@ static int start_job(struct nrf_cloud_fota_job *const job, const bool send_evt)
 			.sec_tag_count = (sec_tag < 0 ? 0 : 1),
 			.pdn_id = 0,
 			.frag_size_override = CONFIG_NRF_CLOUD_FOTA_DOWNLOAD_FRAGMENT_SIZE,
+			.family = AF_INET
```

Comment:
i don't think this is used/needed.

Reply:
It is. It is a newly added field that controls whether IPv6 or IPv4 or both are used. Since we see the IPv6 download fail every time, I added this to select IPv4 only.

Reply:
See https://github.com/nrfconnect/sdk-nrf/blob/c191b6f6c7aeaa48ce842b4ede8238d4d81e7932/subsys/net/lib/download_client/src/download_client.c#L444

Reply:
https://github.com/nrfconnect/sdk-nrf/blob/c191b6f6c7aeaa48ce842b4ede8238d4d81e7932/subsys/net/lib/nrf_cloud/src/nrf_cloud_download.c#L64

do you want to instead add it to `npgps_download_start`?

Reply:
Instead? I was changing it for FOTA, not PGPS.  Or maybe you meant to type `nrf_cloud_download_start` instead of `npgps_download_start`?

Reply:
unless im missing something, the family flag is not used for fota downloads

Reply:
You're right -- I was following the code into `download_client_get` not `fota_download_start_with_image_type`, which eventually calls `fota_download` which creates its own `struct download_client_cfg`.  I think we want to be able to control the family for that as well.